src/dmd/backend/backconfig.d

```diff
@@ -353,6 +354,10 @@ if (SYMDEB_CODEVIEW)
         cod3_set32();
     }
 
+    if (config.objfmt == OBJ_MACH)
+        machDebugSectionsInit();
+    else if (config.objfmt == OBJ_ELF)
+        elfDebugSectionsInit();
```

Comment:
I'm not sure where the best place to put this is

Reply:
Put them in at the beginning of `_initfile`.
You may need to check if `debug_xxx == null`

Reply:
https://github.com/dlang/dmd/search?q=_initfile no results

Reply:
https://github.com/dlang/dmd/blob/f99b447cbc986a0178059d5ff8cd7a2b1443051d/src/dmd/backend/dwarfdbginf.d#L1010

Reply:
Or even better, in a `static this()`.

Reply:
> static this()

Won't work

Reply:
> https://github.com/dlang/dmd/blob/f99b447cbc986a0178059d5ff8cd7a2b1443051d/src/dmd/backend/dwarfdbginf.d#L1010

I thought I tried that. Will give it another shot